### ISSUE TITLE
Hide activity?

### ISSUE THREAD

**Issue by fuzzy76** [gitterHQ/irc-bridge#56]
Is there a way to disable the activity feed from the "gitter" user? If not, would you consider implementing one? For busy projects, it's impossible to follow the actual conversation when it is constantly flooded with commits and issues.

**Comment by mydigitalself** [comment]
Most clients have the ability to mute/ignore a user and this is generally the way other people do it. That work for you?

**Comment by fuzzy76** [comment]
It does work. :) The only downside is that an ignore is global for all channels.

**Comment by lucaswerkmeister** [comment]
> The only downside is that an ignore is global for all channels.

No, that’s client-dependent. In Pidgin, an ignore is per-channel, but also isn’t saved, so I have to ignore each time I log in, which is pretty annoying.

**Comment by fuzzy76** [comment]
Wow, actually did not now that. It's probably been 10 years since I ignored someone (probably on both a different client and operating system that I use now). Weechat allowed me to ignore the "gitter" user on a per channel basis. Still think this feature should have been in irc-bridge somewhere. On the web interface, the repository feed is separate from the chat for a reason.

**Comment by jkaberg** [comment]
Anyone found out howto ignore the gitter bot in Quassel? My rules simply doesn't work.. 

![image](https://cloud.githubusercontent.com/assets/613363/14768388/32205d2c-0a41-11e6-86a0-7815c30102f5.png)

**Comment by julen** [comment]
As a related thought, wouldn't also help to make messages from the gitter user notices? This is how the GitHub bot usually sends messages to channels (it can be adjusted optionally in the webhooks), and IRC clients _at least_ display these messages differently so it'd be easier to follow along conversations in channels.

**Comment by brianjmurrell** [comment]
What about having the activity stream in a separate IRC channel that the user can either join or not?  So each room would have two IRC channels.  One for the main (real people) conversation and antoher for the activity stream if they wished to join that one (or leave it if auto-joined to it when they join the main room channel).
